### PR TITLE
Added GRid Component

### PR DESCRIPTION
**Pull request by ishapatell1**
Screenshot (101)

### CODE REVIEW

File: Components/Grid/grid.html [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+            </h2>
+            <div class = "component-types">
+                <h1 class = "component-name">Grid-2 (50:50) </h1>
+                <h3 class = "component-description">
+                    A two image grid is given by shavi. To copy and use the code click below!
+                </h3>
+                <div class = "component-example">
+                    <div class="grid-50-50-layout">
+                        <div>
+                            <img class="img-round" src="/Images/Gift2.jpg">
```
Comment (iamharnad) [inline Components/Grid/grid.html:55]: use alt attribute in image tags, for better accesibility

File: Components/Grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+                        </div>
+                        <div>
+                            <img class="img-round" src="/Images/Gift2.jpg">
```
Comment (iamharnad) [inline Components/Grid/grid.html:58]: it's good practise to use alt in image tags

File: Components/Grid/grid.html [hunk 3]
```diff
@@ -0,0 +1,144 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>UI-Simple-Grid</title>
+    <link rel = "stylesheet" href = "/style.css">
+    <link rel = "stylesheet" href = "grid.css">
```
Comment (iamharnad) [inline Components/Grid/grid.html:9]: Having only one CSS file is better for the loading-time of your pages, as it means less HTTP requests.

**Review by iamharnad — CHANGES_REQUESTED** [review]

**Review by way2nizam — APPROVED** [review]
Good coding practices used. You can merge it.

**Review by Manpreet-hub — APPROVED** [review]
Looks good to merge!

File: Components/Grid/grid.html (hunk 3 above)
Comment (ishapatell1) [reply Components/Grid/grid.html:9]: I will keep that in mind ..and also as you have suggested using alt in my project.Thankyou!

**Pull request merged**